lib/rules/member-delimiter-style.js

```diff
@@ -13,11 +14,22 @@ const definition = {
     properties: {
         delimiter: { enum: ["none", "semi", "comma"] },
         requireLast: { type: "boolean" },
-        ignoreSingleLine: { type: "boolean" },
+        singleLine: { enum: ["none", "semi", "comma"] },
     },
     additionalProperties: false,
 };
 
+const errorMessages = {
+    unexpected: {
+        comma: "Unexpected separator (,).",
+        semi: "Unexpected separator (;).",
+    },
+    expected: {
+        comma: "Expected a comma.",
+        semi: "Expected a semicolon.",
+    },
+};
```

Comment:
instead making object like this you should consider using `messageId`
https://eslint.org/docs/developer-guide/working-with-rules#messageids

Reply:
i should probably read the docs at some point... 😄